runtime/lua/vim/lsp/util.lua

```diff
@@ -2140,6 +2140,7 @@ end
 ---@param settings table language server settings
 ---@param section  string indicating the field of the settings table
 ---@return table|string The value of settings accessed via section
+---@deprecated
 function M.lookup_section(settings, section)
```

Comment:
I think we need to use `vim.deprecate` now too.

Reply:
Yes, I just checked #26742 and MAINTAIN.md. Thanks